packages/samples/mvp-dapp/src/main.ts

```diff
@@ -42,7 +42,7 @@ async function bootstrap() {
         return _address
       }
 
-      return `http://${_address.address}:${_address.port}`
+      return `http://localhost:${_address.port}`
```

Comment:
`localhost` will be mapped to IPv6 by default when the app is running with Node.js 18